### ISSUE TITLE
Unable to load module file for cuda

### ISSUE THREAD

**Issue by prnk04** [facebookresearch/CodeGen#24]
Hi,

I have been trying to configure CodeGen on WSL2. I have installed cuda toolkit 11.0. However, while running ` module load cuda/11.0` command, I am getting ' ERROR: Unable to locate a modulefile for 'cuda/11.0' ' error.
Is there any dependency that needs to be installed?

![cuda_error](https://user-images.githubusercontent.com/32104339/134545752-2ba28397-508e-447d-a9dd-d46e1e207d4d.png)

**Comment by brozi** [comment]
Actually this command only works on some systems and we shouldn't have included it in install_env.sh.
It should work without it if you have cuda installed and loaded.

**Comment by prnk04** [comment]
I have installed cuda and loaded it. Still, while running command `pip install -v --disable-pip-version-check --no-cache-dir --global-option="--cpp_ext" --global-option="--cuda_ext" ./`, I am getting :
'RuntimeError: Error compiling objects for extension
    Running setup.py install for apex ... error'
However, if I run the above command without --global-option="--cuda_ext", it runs fine. 

I am using torch version: '1.7.1'.
Also, I have set the PATH variable to point to cuda.(/usr/local/cuda/bin:/usr/local/cuda-11.0/bin)
Is there something missing in my configurations?

![configurations](https://user-images.githubusercontent.com/32104339/134646792-2c0f674a-dd0d-4ae4-b594-803a3a459cef.png)
